docs/io/blif_reader.rst

```diff
@@ -0,0 +1,20 @@
+blif reader
```

Comment:
I don't see why we need this extra file. These information could be written, if really necessary, directly in the documenting comment in `blif_reader.hpp` like the other readers. In fact, I think what is already written there is enough, i.e., for a network type to be supported, it has to implement either `create_node` or `create_cover_node`. This implicitly says that currently only `klut` and `cover` are supported (but there could be more in the future).